### PR TITLE
Menu-item component

### PR DESCRIPTION
**Pull request by ShevtsovAnton**
[Trello task - MenuItem component](https://trello.com/c/QCGhE6gJ/43-base-components-menuitem)

### CODE REVIEW

File: components/common/MenuItem/MenuItem.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  align-items: center;
+  padding: 8px;
+  width: 100%;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  color: ${({ active, theme }) => (active ? theme.menu.activeColor : theme.text.defaultColor)};
+
+  @media ${device.tablet} {
+    background: ${({ active }) => (active ? `#333333` : ``)};
```
Comment (OlegSymonchuk) [inline components/common/MenuItem/MenuItem.tsx]: Why not to use a theme color?

File: components/common/MenuItem/MenuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    border-radius: ${({ active }) => (active ? `#333333` : `none`)};
```
Comment (OlegSymonchuk) [inline components/common/MenuItem/MenuItem.tsx]: Are these values valid?

File: components/common/MenuItem/MenuItem.tsx (hunk 1 above)
Comment (ShevtsovAnton) [reply components/common/MenuItem/MenuItem.tsx]: changed to use a theme color

File: components/common/MenuItem/MenuItem.tsx (hunk 2 above)
Comment (ShevtsovAnton) [reply components/common/MenuItem/MenuItem.tsx]: fixed the error

**Review by OlegSymonchuk — APPROVED** [review]

**Pull request merged**